### PR TITLE
P0 card

### PR DESCRIPTION
**Pull request by anshulraheja**
Card Component : 
1. Vertical Card
2. Horizontal Card 
3. Card over Text Media
4. Card with text Only

### CODE REVIEW

File: components/card/card.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                    style="width: 1024px; height: 473px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                    <br><br>
+                    <h2>Horizontal Card</h2>
```
Comment (deekshithmd) [inline components/card/card.html:96]: Try to use span, section wherever possible instead of div

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    gap: 1rem;
+    flex-wrap: wrap;
+    align-items: flex-start;
+}
+
+/* CARDS */
+.card {
+    background: var(--white-color);
+    border-radius: 4px;
+    box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%), 0 1px 3px 0 rgb(0 0 0 / 12%);
```
Comment (PuneetP16) [inline components/card/card.css:10]: Try to use variable instead of direct colors.

**Comment by PuneetP16** [comment]
image
There is much space below the CTA bar.

**Review by PuneetP16 — APPROVED** [review]
Good to merge after necessary improvements.

**Pull request merged**